### ISSUE TITLE
 Web Accessibility Checker' does not work Visual Studio 2022

### ISSUE THREAD

**Issue by rccnw** [MicrosoftDocs/edge-developer#2059]
In section 'External articles and websites', the 'Web Accessibility Checker' item does not work with the most recent version of Visual Studio (and perhaps other previous versions). 

The last update in the GitHub repo is 4 years ago.
see:  https://github.com/madskristensen/WebAccessibilityChecker

On a system where the only version of Visual Studio is VS2022, downloading the extension and running it produces this error message:     'This extension is not installable on any currently installed products'

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: be893a15-4b39-309f-44d2-621f7de455dc
* Version Independent ID: 7d2affa5-ee31-2680-68f8-d2bb6eebbcfd
* Content: [Resources for accessibility testing - Microsoft Edge Development](https://docs.microsoft.com/en-us/microsoft-edge/accessibility/test)
* Content Source: [microsoft-edge/accessibility/test.md](https://github.com/MicrosoftDocs/edge-developer/blob/main/microsoft-edge/accessibility/test.md)
* Product: **microsoft-edge**
* GitHub Login: @MSEdgeTeam
* Microsoft Alias: **msedgedevrel**

**Comment by codepo8** [comment]
I forwarded this to the PM on the VS side of the extension.

**Comment by sayedihashimi** [comment]
Hi @rccnw, have you tried the accessibility checker in the Edge browser? We are thinking to integrate that checker into Visual Stuido 2022. You would get a similar experience to what you get in the extension you linked; you'd just go to a different place to kick that off. I'm wondering if that would meet your needs in this scenario or not? If not, please let us know what it would be missing.

**Comment by rccnw** [comment]
Hi @sayedihashimi -  Are you referring to 'Siteimprove Accessibility Checker'  ?  I installed it on Edge. It seems good, I don't have a background in this topic, so this is my naive take.  Yes, I think it would meet my needs in general.  I don't know how to evaluate a tool like this in relation to standards, but I'd assume it is in accordance and sufficient.

Thanks for the info, my needs are met by this Edge extension.

**Comment by sayedihashimi** [comment]
Hi @rccnw, no sorry I was referring to the built in accessibilty tools. You can find more info at https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/accessibility/accessibility-testing-in-devtools. @codepo8 if there is a better link than what I provided, please add that.

**Comment by rccnw** [comment]
Sorry for the delayed response.  The link you provided to the  'Overview of accessibility testing using DevTools'  page was much appreciated.  That is a great resource that I hadn't discovered on my own, and certainly satisfies my needs.  

Yes, I think an integration with Visual Studio would be great, but at least advertise this resource there so people can find it.

**Comment by captainbrosset** [comment]
Thanks for the discussion, folks. And thanks Sayed for providing this link.
Going back to the original report, should we remove the link to https://marketplace.visualstudio.com/items?itemName=MadsKristensen.WebAccessibilityChecker from this page: https://docs.microsoft.com/en-us/microsoft-edge/accessibility/test#external-articles-and-websites ?
If the extension doesn't work on VS anymore, it seems like we should just remove that link. @sayedihashimi what do you think?

**Comment by sayedihashimi** [comment]
@captainbrosset I have submitted a PR to have the link removed https://github.com/MicrosoftDocs/edge-developer/pull/2091.

**Comment by sayedihashimi** [comment]
@rccnw great, happy to hear that it's working out for you. I'll investigate adding this to VS.
